docs/UserGuide.md

```diff
@@ -251,5 +268,6 @@ Action | Format, Examples
 **Filter** | `filter [m/MODULE] [f/FACULTY] [t/TAG]…`<br> e.g., `filter m/CS2100 t/friends`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **Sort** | `sort [ATTRIBUTE]`<br> e.g., `sort p/`
+**AddMeeting** | `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`<br> e.g., `meeting 1 d/ Meet for finals preperation s/ 12/04/2024 18:00`
```

Comment:
nit: optional and minor, `Add Meeting` with space between

Reply:
Now that i think about it, just ```Meeting``` would be better